rpc/lib/server/http_server.go

```diff
@@ -18,9 +18,23 @@ import (
 	types "github.com/tendermint/tendermint/rpc/lib/types"
 )
 
-// Config is an RPC server configuration.
+// Config is a RPC server configuration.
 type Config struct {
+	// see netutil.LimitListener
 	MaxOpenConnections int
+	// mirrors http.Server#ReadTimeout
+	ReadTimeout time.Duration
+	// mirrors http.Server#WriteTimeout
+	WriteTimeout time.Duration
+}
+
+// DefaultConfig returns a default configuration.
+func DefaultConfig() *Config {
+	return &Config{
+		MaxOpenConnections: 0, // unlimited
+		ReadTimeout:        12 * time.Second,
```

Comment:
Did we need to increase this from 3? 

Also we use 12 for the write timeout but refer to it in comments as 10. So I'll change it to 10 ...

Reply:
No, there was no reason to increase the `ReadTimeout`. That must have slipped (only the write timeout needed to be changed back from 3 to sth. larger than TimeoutBroadcastTxCommit).

Note that while the `WriteTimeout` will be 10 with your changes as mentioned in the comments, it will effectively be 11 now via: https://github.com/tendermint/tendermint/pull/3443/files#diff-380e79adccbaf1276bf6c91cc551cd7fR698

Reply:
Yeh that's fine, so long as the comment is correct (ie. that if broadcast tx is greater than 10 it will cause an increase)

Reply:
The more I think about it the more I think we should probably expose the Read/WriteTimeout directly...

Reply:
> The more I think about it the more I think we should probably expose the Read/WriteTimeout directly...

My refactoring should allow us to do that very easily.